.buildkite/pipeline.yaml

```diff
@@ -0,0 +1,60 @@
+steps:
+  - label: ":eslint: Lint"
+    command:
+      - "yarn install"
+      - "yarn lintwithexclusions"
+    plugins:
+      - docker#v3.0.1:
+          image: "node:10"
+
+#  - label: ":chrome: End-to-End Tests"
+#    command:
+#      # TODO: Remove hacky chmod for BuildKite
+#      - "chmod +x ./scripts/ci/*.sh"
+#      - "chmod +x ./scripts/*"
+#      - "sudo apt-get install build-essential python2.7-dev libffi-dev python-pip python-setuptools sqlite3 libssl-dev python-virtualenv libjpeg-dev libxslt1-dev"
+#      - "./scripts/ci/install-deps.sh"
+#      - "./scripts/ci/end-to-end-tests.sh"
+#    plugins:
+#      - docker#v3.0.1:
+#          image: "node:10"
+
+  - label: ":karma: Tests"
+    command:
+      # Install chrome
+      - "wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -"
+      - "sh -c 'echo \"deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main\" >> /etc/apt/sources.list.d/google.list'"
+      - "apt-get update"
+      - "apt-get install -y google-chrome-stable"
+      # Run tests
+      # TODO: Remove hacky chmod for BuildKite
+      - "chmod +x ./scripts/ci/*.sh"
+      - "chmod +x ./scripts/*"
+      - "./scripts/ci/install-deps.sh"
```

Comment:
Still feels wasteful to duplicate so much of the installation... I can't tell from the BK log how long each step takes right now, since the log get truncated...

Speaking of that, the biggest factor in our log length seems to be Webpack's `--progress` output. Can we change Webpack options used in CI so that the logs shows up start to finish in BK?

Reply:
The log gets truncated for the web view only - as mentioned in the error, there's a download button for when your build explodes and you want to see the whole log. Historically, our stuff breaks before it would reach the point of truncation.